src/Package/Util/Header/Version.php

```diff
@@ -90,14 +90,14 @@ public function fileHasVersionMacro($fname)
     public function getVersionFromHeader()
     {
         $headers = glob($this->package->getSourceDir().DIRECTORY_SEPARATOR.'*.h');
-        $pat = ',define\s+'.$this->macroName.'\s+"(.*)",i';
+        $pat = ',define\s+'.$this->macroName.'\s+(".*"|.*\b),i';
```

Comment:
Well I just mean less characters, regex is never "readable" :)

Shall I change it to your suggestion?